2018-03-28/azblob/zt_examples_test.go

```diff
@@ -477,8 +478,8 @@ func ExampleMetadata_containers() {
 
 	// Create a ContainerURL object that wraps a soon-to-be-created container's URL and a default pipeline.
 	u, _ := url.Parse(fmt.Sprintf("https://%s.blob.core.windows.net/mycontainer", accountName))
-	containerURL := azblob.NewContainerURL(*u,
-		azblob.NewPipeline(azblob.NewSharedKeyCredential(accountName, accountKey), azblob.PipelineOptions{}))
+	credential, _ := azblob.NewSharedKeyCredential(accountName, accountKey)
+	containerURL := azblob.NewContainerURL(*u, azblob.NewPipeline(credential, azblob.PipelineOptions{}))
```

Comment:
given this is an example (and thus a recommended way of using this library) - shouldn't this be checking that the error object returned is nil?

in addition - if `u` is nil here (as the error isn't caught above) there's a panic here?